src/Registration.vue

```diff
@@ -0,0 +1,162 @@
+<!--
+ - @copyright Copyright (c) 2018 Joas Schilling <coding@schilljs.com>
+ - @copyright Copyright (c) 2019 Gary Kim <gary@garykim.dev>
+ -
+ - @author Joas Schilling <coding@schilljs.com>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<template>
+	<div id="terms_of_service">
+		<p v-if="terms.length">
+			<input id="terms_of_service_accepted"
+				v-model="hasSigned"
+				:value="termsId"
+				type="checkbox"
+				name="terms_of_service_accepted"
+				class="checkbox">
+			<label
+				for="terms_of_service_accepted"
+				@click.prevent.stop="showTerms">
+				{{ t('terms_of_service', 'I acknowledge that I have read and agree to the above terms of service') }} ↗
```

Comment:
The `↗` should not be hardcoded, as it will not be shown in the right place in right to left languages. It could be added using CSS classes, something like (untested):
```
#terms_of_service_accepted + label:after {
    content: ` ↗`;
}
```
Of course that will still need to be adjusted for right to left languages, but it should be easier to override with CSS.

Reply:
We currently use this everywhere 👀

Reply:
> We currently use this everywhere

we do? :thinking:

Reply:
https://github.com/nextcloud/server/blob/85e369cddb7b347cccd597ab2f65fc201632091d/apps/theming/templates/settings-admin.php#L141
https://github.com/nextcloud/server/blob/cb68bcef0fcf8c26fb79731031fabfcaf7433f45/core/templates/installation.php#L181
https://github.com/nextcloud/server/blob/8cf7c63a7120f9a7c59e38451fb28d4fe15dcffb/apps/settings/templates/help.php#L27-L32

and more, but we also have the after and it being part of the translation